packages/calypso-e2e/src/types/rest-api-client.types.ts

```diff
@@ -43,16 +43,13 @@ export interface BearerTokenResponse {
 	};
 }
 
-interface SiteMetadata {
-	ID: number;
-	name: string;
-	description: string;
-	URL: string;
-	site_owner: number;
+export interface DomainsMetadata {
+	blog_id: number;
+	domain: string;
```

Comment:
The `/all-domains/` endpoint returns more data, but for E2E purposes the only important one are these.